examples/land-cover-fastai/deep_lulc_walkthrough.ipynb

```diff
@@ -473,7 +479,7 @@
     "    # png_gen('eopatch_{}'.format(idx))\n",
     "    png_gen(op.join(path_out_sampled, 'eopatch_{}'.format(idx)), inference=False)\n",
     "    print(\"Saving RGB, LULC and Inds PNG to eopatch_{} for the coming ML pipeline\".format(idx))\n",
```

Comment:
I would just delete all these printouts, the progress bar is enough, it just spams the notebook

Reply:
agreed, deleted